src/tools/rbd_mirror/Types.cc

```diff
@@ -11,19 +11,19 @@ std::ostream &operator<<(std::ostream &os, const ImageId &image_id) {
             << "id=" << image_id.id;
 }
 
-std::ostream& operator<<(std::ostream& lhs,
-                         const LocalPoolMeta& rhs) {
-  return lhs << "mirror_uuid=" << rhs.mirror_uuid;
+std::ostream& operator<<(std::ostream& os,
+                         const LocalPoolMeta& lpm) {
```

Comment:
It would be nicer to match `local_pool_meta` and `remote_pool_meta` names used in the declarations.

Reply:
Done.